draft-ietf-acme-acme.md

```diff
@@ -303,11 +303,23 @@ JWS objects sent in ACME requests MUST meet the following additional criteria:
 * The JWS MUST NOT have the value "none" in its "alg" field
 * The JWS Protected Header MUST include the following fields:
   * "alg"
-  * "jwk"
+  * "jwk" (only for requests to new-reg and revoke-cert resources)
+  * "kid" (for all requests except for new-reg and revoke-cert requests).
   * "nonce" (defined below)
   * "url" (defined below)
 
-Note that this implies that GET requests are not authenticated.  Servers MUST
+For new-reg requests, there MUST be a "jwk" field containing the key being
+registered, and there MUST NOT be a "kid" field.
+
+For revoke-cert requests that are authenticated by a certificate key, there
+MUST be a "jwk" field containing the public key from the relevant certificate,
+and there MUST NOT be a "kid" field.
```

Comment:
Hm, "authenticated by a certificate key" makes me think:

* This para might be better stated as "For revoke requests, depends on the key being used to authorize.  Issuing / authorized account key => kid, cert key => jwk"
* We should require that if there is a "kid" field present, the server MUST ignore any "jwk" field that is present